extensions/AzureCosmosDBMongoDB/AzureCosmosDBMongoDB/AzureCosmosDBSimilarityType.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+// ReSharper disable InconsistentNaming
+
+namespace Microsoft.KernelMemory.MemoryDb.AzureCosmosDBMongoDB;
+
+public enum AzureCosmosDBSimilarityType
+{
+    COS,
+    IP,
+    L2
```

Comment:
please extend the names for readability, something like `Cosine`, `InnerProduct` etc would be better. 
If different values are needed for serialization, you can use the serialization attributes. Thanks

Reply:
Updated the values.